src/main/java/com/faforever/client/fa/ForgedAllianceService.java

```diff
@@ -36,6 +36,18 @@ public ForgedAllianceService(PreferencesService preferencesService) {
     this.preferencesService = preferencesService;
   }
 
+  public Process startGameOffline(List<String> args) throws IOException {
+    Path executable = getExecutable();
+    List<String> launchCommand = LaunchCommandBuilder.create()
+        .executableDecorator(preferencesService.getPreferences().getForgedAlliance().getExecutableDecorator())
```

Comment:
Why not use `#defaultLaunchCommand()` ?

Reply:
I think there are some options i dont wanna have here, like replay server etc.

Reply:
Have you looked at the function `defaultLaunchCommand()`? I dont see what you mean.

Reply:
will look into it